poi_fr.md

```diff
@@ -0,0 +1,211 @@
+Navitia POI version 0.1
+=======================
+
+# Introduction
+
+Les POI permettent d'enrichir Navitia avec des points de départ ou d'arrivée
+connus des usagers pour la rubrique "Itinéraires" ou la présentation des lieux
+publics ou des services à proximités d'un lieu.
+
+Cette spécification décrit le format d'échange de ces POI entre Fusio et
+Navitia2, et le format d'alimentation cible du nouveau NMM. Certains POI ont
+des impacts sur les résultats d'itinéraires, les effets exacts sont décrits
+dans les spécifications du moteur.
+
+# Format des données
+
+La structure des données est la même que celle utilisée pour le format NTFS
+pour les horaires.  La seule différence est le séparateur ";".
+
+# Liste des fichiers
+Fichier | Contrainte | Commentaire
+--- | --- | ---
+[`poi_type.txt`](#poi_typetxt-requis) | Requis | Ce fichier liste des types de lieux associés aux POI de l'export
+[`poi.txt`](#poitxt-requis) | Requis | Ce fichier contient tous ou partie des POI à ajouter au référentiel
+[`poi_properties.txt`](#poi_propertiestxt-optionnel) | Optionnel | Ce fichier contient les propriétés complémentaires des POI de l'export. Attention, même si ce fichier est optionnel, certaines propriétés ont des impacts directs sur les itinéraires proposés ou le bon fonctionnement de certaines fonctionnalités du NMP.
```

Comment:
```suggestion
[`poi_properties.txt`](#poi_propertiestxt-optionnel) | Optionnel | Ce fichier contient les propriétés complémentaires des POI de l'export. Attention, même si ce fichier est optionnel, certaines propriétés ont des impacts directs sur les itinéraires proposés.
```

Reply:
Done in 961ed4f180b20ef4c1c35f94d18ce6d45b98279a.